infra/core/src/main/java/com/simprints/core/domain/fingerprint/IFingerIdentifier.kt

```diff
@@ -5,15 +5,26 @@ import com.simprints.core.ExcludedFromGeneratedTestCoverageReports
 
 @Keep
 @ExcludedFromGeneratedTestCoverageReports("Enum")
-enum class IFingerIdentifier {
-    RIGHT_5TH_FINGER,
-    RIGHT_4TH_FINGER,
-    RIGHT_3RD_FINGER,
-    RIGHT_INDEX_FINGER,
-    RIGHT_THUMB,
-    LEFT_THUMB,
-    LEFT_INDEX_FINGER,
-    LEFT_3RD_FINGER,
-    LEFT_4TH_FINGER,
-    LEFT_5TH_FINGER,
+// **IMPORTANT**: Do NOT change the order of this enum as it is used in the database by index.
+// Changing the order of the entries in this enum will lead to data corruption or mismatches
+// when retrieving or storing data in the database. Always append new entries at the end.
+enum class IFingerIdentifier(
+    val id: Int,
```

Comment:
I would be very careful about changing the classes in the core/domain as they are likely used directly in the event and sample marshalling and a bunch of other places, adding fields to them at this point might break it. It would be safer to have a dedicated "identifier" enum in the DB module and map values from one to another based on the value and modality.